chain/chunks/src/lib.rs

```diff
@@ -132,43 +137,64 @@ impl RequestPool {
     }
 }
 
+#[derive(Debug, Eq, PartialEq)]
+enum Seal<'a> {
+    Previous,
+    Active(&'a mut ActiveSealChallenge),
```

Comment:
**Previous**: we used to use "previous" to point to a closest predecessor to the object. It always mean that if object has a type T then it predecessor has also type T.

In Seals, we have two different concepts: Active Seals which contain of part_ords and used to challenge Chunk Producer, and Previous Seals that used to remember which Seals we created. Technically, they are separate and serves for different purposes.

That's why using Previous is confusing. I'm not native English speaker and cannot suggest the exact word to replace but it might be better to use something like Inactive / Proven / Used / Passed / Broken, etc.

Reply:
Renamed to `Past` https://github.com/nearprotocol/nearcore/pull/2702/commits/dbc7634dcdaece76119893b6a7c90735d76af048